pageserver/src/http/openapi_spec.yml

```diff
@@ -396,6 +396,9 @@ paths:
         "200":
           description: OK
           content:
+            application/vnd.neon-v2+json:
```

Comment:
Same comment as in the https://github.com/neondatabase/cloud/pull/7553#discussion_r1369109963

I don't think it will work out as is

Reply:
I've now added a version param as per our discussion in that thread.